src/templates/_includes/forms/editableTable.html

```diff
@@ -101,10 +102,12 @@
     {% set jsId = id|namespaceInputId|e('js') %}
     {% set jsName = name|namespaceInputName|e('js') %}
     {% set jsCols = cols|json_encode %}
+    {% set jsMaxRows = maxRows %}
+    {% set jsMinRows = minRows %}
```

Comment:
only set these `js*` values if you have a reason to – notice the other ones either end in `|e('js')` or `|json_encode`, maxing their values safe for JavaScript.